backend/src/app.gateway.ts

```diff
@@ -26,9 +25,16 @@ export class AppGateway
   }
 
   @SubscribeMessage('message')
-  handleMessage(client: Socket, text: string): WsResponse<string> {
+  handleEvent(client: Socket, text: string): WsResponse<string> {
     const message = `Client with id: ${client.id} send a message: ${text}`;
     console.log(message);
     return { event: 'messageToClient', data: message };
   }
+
+  @SubscribeMessage('meeplePlacementMessage')
+  handleMessage(client: Socket, text: string): WsResponse<string> {
+    const message = `Client with id: ${client.id} put a meeple: ${text}`;
+    console.log(message);
+    return { event: 'messageToClientAfterMeeplePlacement', data: message };
```

Comment:
it will be a nice addition to have constants instead of strings like "messageToClientAfterMeeplePlacement" :D